tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -62,3 +80,81 @@ def test_models(
         name_0="hf",
         name_1="vllm",
     )
+
+
+@pytest.mark.parametrize("kv_dtype_n_model",
+                         ["fp8_e5m2" + "#+#" + m for m in E5M2_KV_MODELS] +
+                         ["fp8_e4m3" + "#+#" + m for m in E4M3_KV_MODELS])
```

Comment:
Why not just using a tuple?

Reply:
Because in the test outputs it shows `kv_dtype_n_model0` which is totally not readable and diagnosable. This will show the full string.

Reply:
I know this issue, but I don't think it's worthwhile to use this hacky way. You should be able to easily find out what `kv_dtype_n_model0` means by looking at the test, which you have to do it anyways when you're debugging.

Reply:
Actually, i realize the way to do it, you can just write the key to parameterize as comma separated, and it will parse not as tuple but individual. Thanks